fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/indexes/ValueIndexMaintainer.java

```diff
@@ -75,7 +76,17 @@ public ValueIndexMaintainer(IndexMaintainerState state) {
     @Override
     public RecordCursor<InvalidIndexEntry> validateEntries(@Nullable byte[] continuation,
                                                            @Nullable ScanProperties scanProperties) {
-        return validateOrphanEntries(continuation, scanProperties);
+        if (scanProperties == null) {
+            scanProperties = new ScanProperties(ExecuteProperties.newBuilder()
+                    .setReturnedRowLimit(Integer.MAX_VALUE)
+                    // For orphan index entry validation, it does not hurt to have a weaker isolation.
```

Comment:
Since this does both orphaned entries and missing entries, maybe the comment should just say `// For index entry validation, ...`